### ISSUE TITLE
UI Improvements.

### ISSUE THREAD

**Issue by ghost** [interisti/material-icons-generator-plugin#5]
Instead of a long dropdown list, it would be really nice if the icons were displayed in a grid list with a search bar at the top. Clicking an icon would select it and allow adding multiple icons at once.

The abilitiy for a user preference to set their own default color, size, and dpi variants would be really great.

Another great feature would be to detect the icons already used, set them as checked. If an icon is unchecked it should remove that icon from the drawable folders.

Thanks again for this great plugin!

**Comment by interisti** [comment]
Agree, right now searching is really pain, i was thinking about making window like nuget packages have. pretty much what you are suggesting in first part. so i will definitely make this changes as soon i will have some free time.

detecting and removing would also be great, so after i finish first part i will also make some time for this also.

btw i've uploaded this plugin to public addin repo so you can install/update it within XS.

Thanks for your feedback.

**Comment by interisti** [comment]
Hi @robertbaker 

I've separated VS version of the plugin (https://github.com/interisti/vs-material-icons-generator)

And made some enhancements to UI like: Preview, Searching, VS Styling.

You can check it here: https://visualstudiogallery.msdn.microsoft.com/3741d776-755b-4002-a6be-e86fa605a315
